adafruit_displayio_ssd1305.py

```diff
@@ -65,7 +71,7 @@ class SSD1305(displayio.Display):
         One of (0, 90, 180, 270)
     """
 
-    def __init__(self, bus, **kwargs):
+    def __init__(self, bus: Union[displayio.Fourwire, I2C], **kwargs) -> None:
```

Comment:
Why did you type it as `I2C`, is there a use case for it somewhere?

Reply:
displayio.I2CDisplay instead?

Reply:
yeah, after checking the sample code for simpletest that makes more sense